README.md

```diff
@@ -16,72 +16,72 @@ Backed by Y Combinator
 
 ---
 
-**Fern is an open source format for defining REST APIs.**
-You can think of it like a programming language to describe
-your API: your endpoints, types, errors, and examples.
+# Fern
 
-This repository contains the **Fern compiler.** The compiler transforms the API description into useful outputs, like:
+**Fern** is an **open source toolset** for **defining, building and generating APIs and SDKs**. Fern can be used in your existing projects through OpenAPI, and integrate effortlessly into your existing workflows. It can also serve as the basis of your API development, by describing your endpoints in a Fern specification, and having Fern generate your API backend code for you.
 
-### 🌿 SDKs
+This repository contains the Fern **compiler**. The compiler transforms an API definition, either in Fern or OpenAPI format into various code artifacts, such as Client Libraries, SDKs, or even fully-fledged **Express** or **FastAPI** backends.
```

Comment:
should we use `specification` to be consistent?

Reply:
Call API definitions specifications everywhere you mean?

Reply:
I like definition better than specification.

Reply:
The reasoning why is b/c API Definition refers to the actual API (yaml files in Fern, or openapi.yml) and specification refers to the schema of the definition (i.e. what things can the definition be about).

Reply:
Therefore: the Fern schema is a specification distinct from the OpenAPI specification, but a specific API defined using either specification would be a definition of said API?

Reply:
thats right ^